protostar/testing/environments/fuzz_test_execution_environment.py

```diff
@@ -171,12 +172,15 @@ async def test(**inputs: Any):
                 self.fork_state_for_test()
                 self.set_cheatcodes_for_test()
 
+                is_first_run = runs_counter_initial_value == runs_counter.count
                 run_no = next(runs_counter)
                 with self.state.output_recorder.redirect(("test", run_no)):
                     with with_reporter(protostar_reporter):
                         try:
                             this_run_resources = await self.execute_test_case(
-                                function_name, **inputs
+                                function_name=function_name,
+                                calculate_estimated_fee=is_first_run,
```

Comment:
This might prove pretty useless when we want to detect price spikes in some corner cases of the fuzzed inputs

Reply:
in [new approach](https://github.com/software-mansion/protostar/pull/1013/commits/3efa247b9c0b19218279da5b017f02f38aff4ebf) I changed the mechanism of fee estimation for fuzzing. Now, the biggest fee encountered is shown in the summary.